src/test/unittest/ut.c

```diff
@@ -117,6 +117,20 @@ ut_strerror(int errnum, char *buff, size_t bufflen)
 				strcpy_s(buff, bufflen, UNMAPPED_STR);
 	}
 }
+
+/*
+ * ut_new_process -- creates and executes new synchronous process
+ */
+intptr_t
+ut_new_process(int argc, const char *argv[])
+{
+	int new_size;
+	char **cmd = ut_append_to_array(argc, argv, NULL, &new_size);
```

Comment:
Is there a reason you need the last element to be NULL? Please write so in a comment.